### DISCUSSION TITLE
Suggestion - Use monospaced font in DX Cluster Console view

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#161] [Ideas]
Hi Ladislav,
The view of the console telnet console window would be better if the user was given a choice selecting monospaced fonts for the view of the telnet cluster raw data flow.

73 de LB2EG Richard

![Screenshot from 2023-01-15 23-57-27](https://user-images.githubusercontent.com/22942440/212572317-b6a2ca43-18a7-41a7-a9cd-1ecf1b452251.png)

**Comment by foldynl** [comment]
Yes, you are right. The monospaced font is needed for DXC. I solved it about a year ago. At that time I had a problem with the implementation under WIN (it didn't look nice), so I postponed it. I'll try to get back to it.

**Reply by rsaue** [reply]
What if you keep the today font as standard, but gives the user an option to change to a font that is provided with the operating system?

73 de LB2EG Richard

**Reply by foldynl** [reply]
I will try another approach to this problem. Possible zooming of the monospace font in the DXC window. 

To be honest, I don't like applications that have settings for everything. It means that the author did not know what the user wanted and that's why many thinks must be parameterized. I would like to provide an application that has settings only for really important things.

**Reply by rsaue** [reply]
I would not put the blame on the Developer. 

User settings may be looked upon as a tool for the developer to adress the diversity in the Ham community, as the average SSB ragchew ham does not use a log program in the same way as a  ham who is into DX-ing.

 If you predefine the settings to fit to - say - 75 %  of the QLog user community, the average ham rarely will need to look into the settings.

Only the remaining 25 % of the users will look for the "advanced settings" Tab, and they usually know what they are looking for :)

Richard

**Reply by foldynl** [reply]
yes I agree with you. 

I also belong to the community that is looking for "advanced setting" (that's why I also use Linux, where I can set everything as I need). On the other hand, it is needed to build "well-defined" user interface. And that's the problem with many applications. Assuming to make all user's settings accessible, applications become quite confusing. Then, unfortunately, those applications are built for the 25% of users (even if I think the percentage is lower).

let's go back to the DXCC issue. Yesterday I found, in my opinion, an interesting solution to the issue. Of course, It is necessary to test it on WIN, but the thinking is like that. Monospace font can be used. Its size can be changed directly in the window using CTRL + mouse wheel. The final size will be remembered for the next session. I think that if it is clearly described in the WIKI, then it is an elegant solution to this issue.
